src/common/gres.c

```diff
@@ -13691,7 +13702,8 @@ extern char *gres_flags2str(uint8_t config_flags)
  */
 extern void add_gres_to_list(List gres_list, char *name, uint64_t device_cnt,
 			     int cpu_cnt, char *cpu_aff_abs_range,
-			     char *device_file, char *type, char *links)
+			     bitstr_t *cpu_aff_mac_bitstr, char *device_file,
```

Comment:
I'd prefer if this was added at the end, on a new line, to have simpler merge/patch in the  future. There does not seem to be a reason to add it in the middle of the function argument list.